### ISSUE TITLE
Slide List not scoped correctly

### ISSUE THREAD

**Issue by evanblack** [buildinternet/supersized#17]
Because the variable vars.slide_list is never defined, the scope doesn't exist and the plugin grabs every li element on the page. 

Slide list should be declared under the variables:

```
$.supersized.vars = {

    // Elements
    slide_list          :   '#slide-list'

};
```

**Comment by jeanmichel71** [comment]
Same issue here with v3.2.1, I had to set slide_links option to false.
Line 246: $('li', vars.slide_list).click(function(){ ... }
kills all my [li][a href] links

**Comment by jeanmichel71** [comment]
in jQuery documentation [http://docs.jquery.com/%24], context should be a DOM element or jQuery object, not a CSS selector

**Comment by samdunn** [comment]
Thanks for pointing this out guys - this should be resolved in 3.2.3 which i just pushed through
